v2.6/getting-started/kubernetes/installation/hosted/canal/1.7/rbac.yaml

```diff
@@ -0,0 +1,114 @@
+# Calico Roles
+# Pulled from https://docs.projectcalico.org/v2.5/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
```

Comment:
Do we need the `1.7/`  in the path for these files?

Reply:
I don't think it is necessarily needed but I thought it would helps us remain consistent and avoid a move when we have to add a new versioned manifest.

Reply:
Yeah, it's just that v1.7 is way old now.

I'd rather add those in later when we find a reason to - I'd like to remove them from our other directories as well but it's a breaking change, so haven't yet.

Since we're adding a new file, I'd suggest not putting it in a versioned directory.

Reply:
This question is why I had tried to add a doc about how we handle manifests so we didn't need to ask these questions. (https://github.com/projectcalico/calico/pull/1175)

Reply:
I know there is a logic to how we are handling the manifests but I would tend to agree with @caseydavenport that it is a bit confusing.